napari/_tests/test_viewer.py

```diff
@@ -85,7 +91,7 @@ def test_viewer(make_napari_viewer):
     'Tracks': 0,
     'Points': 8,
     'Labels': 14,
-    'Shapes': 17,
+    'Shapes': 19,
```

Comment:
There are 19 methods in the `napari.layers.shapes._shapes_key_bindings.py` file.  There are 2 methods that were not assigned a shortcut, so before, those were not counted here.  They are registered with the action_manager though, so they are counted now.

Reply:
Which two methods are those, curious if there was something funny going on which we might need to fix?

Reply:
they are `move_shapes_selection_to_front` and `move_shapes_selection_to_back`.  
https://github.com/napari/napari/blob/472ea41d0724b6d457ee033b3c5aecb87e13b460/napari/layers/shapes/_shapes_key_bindings.py#L149

https://github.com/napari/napari/blob/472ea41d0724b6d457ee033b3c5aecb87e13b460/napari/layers/shapes/_shapes_key_bindings.py#L154

Reply:
Ha! I guess we forgot/ never got around to adding a keybinding. Now people can choose what they want to add themselves!!

Reply:
Give 'em a little fun.  😆